jena-fuseki2/jena-fuseki-ui/package.json

```diff
@@ -61,7 +63,7 @@
     "sinon": "^17.0.0",
     "vite": "^5.0.12",
     "vite-plugin-istanbul": "^5.0.0",
-    "vitest": "^0.34.6",
+    "vitest": "^1.3.1",
```

Comment:
I wonder why dependabot didn't pick this up. Maybe because it was a major update.

Reply:
I hope to be able to replace istabul by vitest as vite - written by same creator of Vue - is becoming more widespread, even among React and Angular projects, and more and more tools too.